.github/workflows/releases.yml

```diff
@@ -0,0 +1,28 @@
+# A GitHub action that notifies the developer
+# changelog repository of any new releases.
+
+name: Notify changelog
+
+on:
+  # Only trigger for a full release,
+  # ignoring pre-releases and drafts
+  release:
+    types:
+      - released
+
+jobs:
+  notify:
+    # This job can run on the latest Ubuntu
+    # and it should not take more than 3 minutes
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+
+    steps:
+      # There's really only 1 step, and i
+      - name: Notify changelog of new release
+        uses: peter-evans/repository-dispatch@v1
```

Comment:
Does this need to be added as a dev dependency to this repository?

Reply:
No, GitHub actions will install this on-demand in their CI pipeline. This does require us to enable the use of these kind of actions in the settings. Please see the **Things we need to do now** section of my PR description for more detauls.

Reply:
Should this be a fork of `peter-evans/repository-dispatch@v1` in case this where to change in the future and break the build?